app.py

```diff
@@ -1593,6 +1608,14 @@ def sendEmail(project_id, evaluation_name, show_score):
     """
     project = Permission.query.filter_by(project_id=project_id).first()
     path_to_load_project = "{}/{}/{}".format(base_directory, project.owner, project.project)
+
+    #################
+    path_to_project_includeEvaluationFolder = "{}/{}/{}/evalutaion".format(base_directory, project.owner, project.project)    #make directory to place each evaluation under a file named after eva_name
```

Comment:
"evalutaion" is spelled wrong

Reply:
I actually think we should change the name of this folder anyway. Let's call it

evaluation_feedback